examples/use-cases/Kernel2Kernel&Vfio2Noop/netsvc.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: networkservicemesh.io/v1
 kind: NetworkService
 metadata:
-  name: kernel2kernel-vfio2noop
```

Comment:
Why was this name changed?

Reply:
Because it is the specificity of the test.
There are 2 endpoints, and each of them provides different services. This one provides only `kernel2kernel`